lucille-core/src/main/java/com/kmwllc/lucille/stage/ComputeFieldSize.java

```diff
@@ -0,0 +1,45 @@
+package com.kmwllc.lucille.stage;
+
+import java.util.Iterator;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import com.kmwllc.lucille.core.*;
+
+/**
+ * Computes sizes of given byte array field and adds a field with that size
+ *
+ * Config Paramters:
```

Comment:
Formatting like this in javadoc generally requires html-like markup to render properly.